recipe/meta.yaml

```diff
@@ -10,19 +10,19 @@ source:
   sha256: e2714f07b750458beaf5b07b670ea7b6058ee155c021c987d0b8e6a40bf3446f
 
 build:
-  noarch: python
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - python >=3.7
+    - python
```

Comment:
Is this just because there are no new Python 3.6 builds?

Reply:
This is because we are no longer `noarch: python`.  There is now a separate build for each python version, and the particular python version is not controlled with this kind of a constraint.  Instead, we would need to skip an unsupported python version e.g. with `skip: true # [py<37]`.  But that's not not necessary because we don't build python less than 3.8 anyway.

Reply:
I missed the Python 3.7 drop (on conda-forge). Seems fine to me, just didn't know it.

I'm somewhat worried that, tomorrow, the team that develops this library and others bumps up the proto-plus requirement and this ends up complicating the recipe in a way we eventually want to "decomplicate". Let me try to reach out (I work at Google but make no promises).